.vib/mediawiki/cypress/cypress/integration/mediawiki_spec.js

```diff
@@ -0,0 +1,78 @@
+/// <reference types="cypress" />
+import { random, getPageUrlFromTitle, confirmLogOut } from './utils';
+
+it('allows to log in and out', () => {
+  cy.login();
+  cy.get("li[id*='preferences']");
+  cy.contains('Log out').click();
+  confirmLogOut();
+  cy.contains('You are now logged out');
+});
+
+it('allows to create user', () => {
+  cy.visit('/index.php?title=Special:CreateAccount');
+  cy.fixture('users').then((user) => {
+    cy.get('input[id="wpName2"]').type(`${user.newUser.username}-${random}`);
+    cy.get('input[id="wpPassword2"]').type(
+      `${user.newUser.password}-${random}`
+    );
+    cy.get('input[id="wpRetype"]').type(`${user.newUser.password}-${random}`);
+  });
+  cy.contains('button', 'Create your account').click();
+  cy.fixture('users').then((user) => {
+    cy.contains(`Welcome, ${user.newUser.username}-${random}`);
```

Comment:
```suggestion
    cy.contains('Welcome', ${user.newUser.username}-${random});
```

Reply:
As discussed in a private call, this approach is not valid, as `cy.contains()` doesn't allow to have both arguments as content. Ref: [cy.contains() doc](https://docs.cypress.io/api/commands/contains#Syntax)